src/main/java/com/javadeobfuscator/deobfuscator/transformers/general/peephole/UnconditionalSwitchRemover.java

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
So this normalizer fixes Zelix string decryption, by changing 
new int[]{1,2,3}
to
int[0] = 1;
int[1] = 2;
Should we include as part of Peephole optimizer?

Reply:
If the side effects aren't important to decompilation and understanding how the code works, then leave it. If the side effects are misleading (i.e. makes parts of code appear to be run when they're not), then we should fix that before merging